pkg/grpc/device_definition.proto

```diff
@@ -283,6 +308,7 @@ service DeviceDefinitionService {
 	rpc GetDeviceMakeByName(GetDeviceMakeByNameRequest) returns (DeviceMake);
 	rpc GetDeviceMakes(google.protobuf.Empty) returns (GetDeviceMakeResponse);
 	rpc GetFilteredDeviceDefinition(FilterDeviceDefinitionRequest) returns (GetFilteredDeviceDefinitionsResponse);
+	rpc GetDeviceCompatibility(GetDeviceCompatibilityListRequest) returns (GetDeviceCompatibilityListResponse);
```

Comment:
I wonder if the name conveys well enough what this does. When I see `GetDeviceCompatibility` I'm thinking singular.